shell/platform/android/io/flutter/embedding/engine/FlutterEngine.java

```diff
@@ -101,6 +101,11 @@ public void onPreEngineRestart() {
 
           platformViewsController.onPreEngineRestart();
         }
+
+        @Override
+        public void onDisplayPlatformView(int viewId, int x, int y, int width, int height) {
```

Comment:
Actually, we won't need this method here.  Instead call `flutterJni.setPlatformViewsController(platformViewsController)` below `flutterJNI.addEngineLifecycleListener(engineLifecycleListener);`.